aea/cli/launch.py

```diff
@@ -100,29 +81,5 @@ def _launch_subprocesses(click_context: click.Context, agents: List[Path]) -> in
     for agent in launcher.not_failed:
         logger.info(f"Agent {agent} terminated with exit code 0")
 
-    return launcher.num_failed
-
-
-def _launch_threads(agents: List[Path]) -> int:
-    """
-    Launch many agents, multithreaded.
-
-    :param click_context: the click context.
-    :param agents: list of paths to agent projects.
-    :return: exit status
-    """
-    aeas = []  # type: List[AEA]
-    for agent_directory in agents:
-        with cd(agent_directory):
-            aeas.append(AEABuilder.from_aea_project(".").build())
-    runner = AEARunner(
-        agents=aeas, mode="threaded", fail_policy=ExecutorExceptionPolicies.log_only
-    )
-    try:
-        runner.start(threaded=True)
-        runner.join_thread()  # for some reason on windows and python 3.7/3.7 keyboard interuption exception gets lost so run in threaded mode to catch keyboard interruped
-    except KeyboardInterrupt:
-        logger.info("Keyboard interrupt detected.")
-    finally:
-        runner.stop()
-    return runner.num_failed
+    logger.debug(f"Exit cli. code: {launcher.num_failed}")
+    sys.exit(launcher.num_failed)
```

Comment:
I think this should be upadted. I think this is not a good practice. 

Say there is a failure in our system such that all the agents fail for the same reason (e.g. no networking).We return a different error code depending on the number of agents. Why not just `1`? This could potentially overlap with other error codes, e.g. `2` in UNIX has a special meaning, misuse of command line syntax.

Reply:
Yes let's return `1` indeed. I agree.

Reply:
done